### ISSUE TITLE
NPE when omitting field in destination

### ISSUE THREAD

**Issue by schuettec** [remondis-it/remap#61]
Given the following scenario:
```
public class A {

  public String multiMapped;
  // Getters & Setters...
}

public class AResource {

  public String multiMapped1;
  public String multiMapped2;
  public String multiMapped3;
  public String omitted;
}

public class MapperTest {

  @Test
  public void shouldMap() {
    Mapping.from(A.class)
        .to(AResource.class)
        .omitInDestination(AResource::getOmitted)
        .replace(A::getMultiMapped, AResource::getMultiMapped1)
        .withSkipWhenNull(Function.identity())
        .replace(A::getMultiMapped, AResource::getMultiMapped2)
        .withSkipWhenNull(Function.identity())
        .replace(A::getMultiMapped, AResource::getMultiMapped3)
        .withSkipWhenNull(Function.identity())
        .mapper();
  }
}

```
...the Mapper will throw an NPE. This is because `com.remondis.remap.Mapping.denyAlreadyOmittedProperty(PropertyDescriptor)` erroneously relies on not having an `omitInDestination` in the mapping configuration.

**Comment by schuettec** [comment]
Temporary workaround is to move the `omitInDestination` to the bottom of the mapping configuration.

**Comment by schuettec** [comment]
Fixed with #62
